build.cake

```diff
@@ -50,11 +50,7 @@ Setup(() =>
 //////////////////////////////////////////////////////////////////////
 
 Task("Clean")
-    .Does(() =>
-{
-    CleanDirectories(new DirectoryPath[] {
-        buildResultDir, binDir, testResultsDir, nugetRoot});
-});
+    .Does(() => CleanDirectories(parameters.Paths.Directories.ToClean));
```

Comment:
Just trying to optimize my keystrokes before they run out :smile:  But I'll adjust, as I'm good with both :wink: